wagtailio/areweheadlessyet/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+from wagtail.admin.edit_handlers import FieldPanel, MultiFieldPanel
+from wagtail.core.fields import RichTextField
+from wagtail.core.models import Page
+
+
+class AreWeHeadlessYetHomePage(Page):
+    THUMBS_UP = "\U0001f44D"
+    THUMBS_DOWN = "\U0001f44E"
+    ICON_CHOICES = [
+        (THUMBS_UP, THUMBS_UP),
+        (THUMBS_DOWN, THUMBS_DOWN),
+    ]
```

Comment:
I think we should store icon choices with string identifiers (`thumbsup`, `thumbsdown`) rather than emoji codes, and leave it up to the front-end to render either vanilla emojis or image files.

Reply:
Alright, I've updated it. Is that what you meant?

Reply:
Yep!